css/style.css

```diff
@@ -0,0 +1,283 @@
+html {
+  box-sizing: border-box;
+  width: 100%;
+}
+body{
+	background-color: white;
+	margin: 0 auto;
+	width: 100%;
+}
+header{
+	width: 100%;
+	height: 120px;
+	border-bottom: solid yellow 4px;
+	background-color: black;
+	color: white;
+}
+ul{
+	display: inline-block;
+	float: right;
+}
+ul .btn{
+	display: none;
+	cursor: pointer;
+}
+ul li {
+	float: left;
+}
+
+ul li a{
+	display: block;
+	padding: 15px;
+}
+.headline {
+	background-color: #D8DCDC;
+	height: 43px;
+	width: 100%;
+	color: black;
+	text-align: center;
+	padding: 20px 0 15px 0;
+}
+.main{
+	background: url(../images/coworking.jpg) center center no-repeat;
+	background-size: cover;
+	width: 100%;
+	height: 650px;
+	overflow: hidden;
+	box-shadow: inset 0px 0px 20px rgba(0,0,0,.8);
+	margin: 0 auto;
+/*	background-image: url("../images/coworking.jpg")*/
+}
+/*img{
+	position: absolute;
+	z-index: -1;
+
+}*/
+.overlay{
+	width: 350px;
+	height: 500px;
```

Comment:
try avoiding explicit heights on divs
content should determine the height automatically